pkgs/misc/cups/default.nix

```diff
@@ -41,6 +41,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "lib" "dev" "man" ];
 
+  patches = lib.optional stdenv.isDarwin ./0001-TargetConditionals.patch;
```

Comment:
Does the patch break Linux? If not please apply it unconditionally.

Reply:
I don't have a way to test handy. I thought it might avoid unnecessary rebuilds.

Reply:
It shouldn't affect it though, since it's guarded by an `ifdef`.